datasets/doqa/doqa.py

```diff
@@ -0,0 +1,184 @@
+            }),
+                "followup": nlp.Value("string"),
+                "yesno": nlp.Value("string"),
+                "orig_answer": nlp.features.Sequence({
+                    "text": nlp.Value("string"),
+                    "answer_start": nlp.Value("int32"),
+            }),
+            }),
+            homepage="http://ixa.eus/node/12931",
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        path = dl_manager.download_and_extract(_URL)
+        if self.config.name == 'cooking':
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset",  "doqa-cooking-test-v2.1.json")},
+                ),
+                nlp.SplitGenerator(
+                    name=nlp.Split.VALIDATION,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset", "doqa-cooking-dev-v2.1.json")},
+                ),
+                nlp.SplitGenerator(
+                    name=nlp.Split.TRAIN,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1","doqa_dataset",  "doqa-cooking-train-v2.1.json")},
+                )
+            ]
+        elif self.config.name == "movies":
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset",  "doqa-movies-test-v2.1.json")},
+                )
+            ]
+        elif self.config.name == "travel":
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset", "doqa-travel-test-v2.1.json")},
+                )
+            ]
+
+    def _generate_examples(self, filepath):
+        """Yields examples."""
+        
+        with open(filepath) as f:
+            data = json.load(f)
+            for row in data["data"]:
+                title = row["title"]
+                background = row["background"]
+                paragraphs = row["paragraphs"]
+                for p in paragraphs:
+                    context = p["context"]
+                    id_ = p["id"]
+                    qas = p["qas"]
+                    for qa in qas:
+                        question = qa["question"]
+                        answers = qa["answers"]
+                        id1 = qa["id"]
+                        yesno = qa["yesno"]
+                        followup = qa["followup"]
+                        answer_text = [answer["text"] for answer in answers]
+                        answer_start = [answer["answer_start"] for answer in answers]
+                        
+                        orig_answer_start = [qa["orig_answer"]["answer_start"]]
+                        orig_answer_text = [qa["orig_answer"]["text"]]
```

Comment:
Those two fields are supposed to be `Sequence` according to the features description, but it looks like they always contain exactly one element. Is this intended ?